MarsQAOnboarding/LanguageFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using TechTalk.SpecFlow;
+using MarsOnboarding.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace MarsOnboarding
+{
+    [Binding]
+    public class LanguageFeatureStepDefinitions
+    {
+        [Given(@"User is logged into localhost successfully")]
+        public void GivenUserIsLoggedIntoLocalhostSuccessfully()
+        {
+            Login_Page loginpage = new Login_Page();
+            loginpage.LoginSteps();
+        }
+
+        [When(@"Adding new '([^']*)' and '([^']*)' to the language list")]
+        public void WhenAddingNewAndToTheLanguageList(string language, string level)
+        {
+            Language_Page languagepage = new Language_Page();
```

Comment:
you should move all your creation of objects to the constructor, e.g   Language_Page languagepage = new Language_Page(); should in constructor see Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors